client/public/static/js/verification.js

```diff
@@ -90,24 +83,28 @@ $(document).ready(function () {
     };
 
     fetch(RESEND_VERIFICATION_URL, {
-      method: PUT_ACTION,
```

Comment:
You can leave this as PUT_ACTION, PUT_ACTION is defined as PUT in the constants.js file.